ground/gcs/src/plugins/opmap/opmapgadgetwidget.cpp

```diff
@@ -98,8 +98,8 @@ OPMapGadgetWidget::OPMapGadgetWidget(QWidget *parent) : QWidget(parent)
     m_mouse_waypoint = NULL;
 
 	pm = NULL;
-	obm = NULL;
-	obum = NULL;
+    uavo_mgr = NULL;
```

Comment:
After this PR is merged, could you make a new PR that reinvents this whole file to spaces? It is currently mixed and makes it look like you messed up indentation in PR (I checked, you are using spaces appropriately)